src/status_im/extensions/capacities/subs.cljs

```diff
@@ -0,0 +1,101 @@
+(ns status-im.extensions.capacities.subs
+  (:require [re-frame.core :as re-frame]
+            [status-im.utils.ethereum.tokens :as tokens]
+            [status-im.utils.ethereum.core :as ethereum]
+            [status-im.utils.money :as money]))
+
+(re-frame/reg-sub
+ :extensions/identity
+ (fn [_ [_ _ {:keys [value]}]]
+   value))
+
+(defn get-token-for [network all-tokens token]
+  (if (= token "ETH")
+    {:decimals 18
+     :address  "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee"}
```

Comment:
maybe move map ` {:decimals 18
     :address  "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee"} ` to constant def